server/routes/rootRouter.js

```diff
@@ -1,10 +1,11 @@
 import express from "express"
 import clientRouter from "./clientRouter.js"
+import petTypesRouter from "./api/v1/petTypesRouter.js"
 
 const rootRouter = new express.Router()
 
-rootRouter.use()
+rootRouter.use("/api/v1/pets", petTypesRouter)
 
 rootRouter.use("/", clientRouter)
 
```

Comment:
Now on line 8 we can add the following code which will handle any requests to `"/"` and send a redirect as the response.  This will trigger a request to be made to `"/pets"` and then that will be handled by the clientRouter

```js
rootRouter.get("/", (req, res) => {
  return res.redirect("/pets")
})
```

Reply:
We made these suggested edits! Thank you!